core/src/snapshot_packager_service.rs

```diff
@@ -211,6 +211,21 @@ impl SnapshotPackagerService {
             start.elapsed(),
         );
 
+        info!("Saving obsolete accounts...");
+        let start = Instant::now();
+        let result = snapshot_utils::write_obsolete_accounts_to_snapshot(
+            &bank_snapshot_dir,
+            &state.snapshot_storages,
+            state.snapshot_slot,
+        );
+        if let Err(err) = result {
+            warn!("Failed to serialize obsolete accounts: {err}");
+            // If serializing the obsolete accounts failed, we do *NOT* want to mark the bank snapshot
+            // as loadable so return early.
+            return;
+        }
+        info!("Saving obsolete accounts... Done in {:?}", start.elapsed(),);
```

Comment:
nit nit: trailing comma in macro

```suggestion
        info!("Saving obsolete accounts... Done in {:?}", start.elapsed());
```